### ISSUE TITLE
check the list of fields added to "all"

### ISSUE THREAD

**Issue by newgene** [biothings/mychem.info#55]
For example, `drugbank.id` field should be included in "all",  but does not seems included:

Either http://mychem.info/v1/chem/DB01076 or http://mychem.info/v1/query?q=drugbank.id:DB01076 works, but not this:

http://mychem.info/v1/query?q=DB01076

We should double check all fields need to be included in "all".

**Comment by newgene** [comment]
Let's add some nosetests for these query cases.

Will close this after we test on the next data release.

**Comment by newgene** [comment]
More fields added to "all":

```
ndc.substancename
aeolus.drug_name
chembl.pref_name
ginas.preferred_name
unii.preferred_term
```

**Comment by sirloon** [comment]
done in b7082fae0ef714a7b40547059ca2b6a387a45a13

**Comment by sirloon** [comment]
tests implemented in 8c4f94ce1301b1b32006dcc9d4171d84b7cee69f
